components/web/gwsocket/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Adam Stevko
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		gwsocket
+COMPONENT_VERSION=	0.1
+COMPONENT_SUMMARY=	fast, standalone, language-agnostic WebSocket server RFC6455 compliant
+COMPONENT_PROJECT_URL=	http://gwsocket.io/
+COMPONENT_FMRI=		web/gwsocket
+COMPONENT_CLASSIFICATION=Web Services/Application and Web Servers
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	v$(COMPONENT_VERSION).tar.gz
```

Comment:
Please,  rename $(COMPONENT_ARCHIVE) so that it can be stored in USERLAN_ARCHIVES with some unique name.